neovim/api/common.py

```diff
@@ -69,8 +69,8 @@ class RemoteMap(object):
     It is used to provide a dict-like API to vim variables and options.
     """
 
-    def __init__(self, obj, get_method, set_method=None, self_obj=None):
-        """Initialize a RemoteMap with session, getter/setter and self_obj."""
```

Comment:
I'm not sure if this is used or not, but it seems like there's no test coverage. If it is used a test would be nice.

Reply:
Thank you for your comment. As you say, I should add a test covering this change. 
I'll add several tests in test/test_common.py.

Reply:
Just one test is enough. @bfredl would know better whether `self_obj` is still needed.

Reply:
`self_obj` looks completely dead to me.

Reply:
@bfredl 

Thank you for your check!

I've checked existing test code(under `test` directory), and I have a question.

`RemoteSequence`/ `RemoteMap` isn't public, and their `__init__` methods(fixed by this PR) are called internally. On the other hand, it seems that current test code covers public API. 
I wonder whether it's better to also cover private APIs(In this case, `__init__`  of `RemoteSequence`/ `RemoteMap`). 

Best regards.

Reply:
I don't think tests of internal classes are needed, if both classes are already covered by public API, there is no need to add a test.

Reply:
OK, Thank you or your comment!